Makefile

```diff
@@ -7,7 +7,9 @@ COMMIT := $(shell git rev-parse HEAD)
 VERSION := $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
 
 # Image tag: if image tag is not set, set it with version (or short commit if empty)
+ifeq (${IMAGE_TAG},)
```

Comment:
without this, the IMAGE_TAG even if specified is always set to the short commit.